### DISCUSSION TITLE
Github Copilot in IntelliJ IDEA has problem

### DISCUSSION THREAD

**Discussion by civetming** [community/community#7130] [Copilot Conversations] [answered]
I use Github Copilot in IntelliJ IDEA，Login to github no response
![image](https://user-images.githubusercontent.com/44012892/139783586-3083be08-20a8-4b72-8635-100a8010bce1.png)

![image](https://user-images.githubusercontent.com/44012892/139783714-cfefc30f-d778-415e-8efb-398d43796bf7.png)

**Comment by jansorg** [comment, marked as answer]
This seems like a problem with your network connectivity. Are you able to open https://github.com/login/device/ in a browser? Do you have configured a proxy or firewall for IntelliJ IDEA?

**Reply by zhangyi1357** [reply]
I‘ve solved this problem by checking my proxy in IDEA. Without proxy IDEA can't access github.com!
Thanks a lot for your answer!

**Comment by zhuxiaoker** [comment]
same issue. how to fix it?

**Reply by jansorg** [reply]
As above, please verify your proxy settings of your IDE. It needs to access github.com to authenticate with your GitHub account.

**Reply by Symbolk** [reply]
Same issue here, I make sure that no proxy is set, I have access to google and github.com under a global system shadowsocks proxy, and have tried both the wifi and 5G.
Is there any chance that it is an internal problem?
